rules/handle-unhappy-paths/rule.md

```diff
@@ -0,0 +1,34 @@
+---
+seoDescription: Ensure your code handles all possible scenarios, including edge cases, to improve robustness and reliability.
+type: rule
+title: Code - Do you handle all possible scenarios?
+uri: handle-unhappy-paths
+authors:
+  - title: Brady Stroud
+    url: https://www.ssw.com.au/people/brady-stroud
+related: []
```

Comment:
Hey @bradystroud 

- [ ] Can you do a search for SSW Rules on validation and error handling and see if there is something to be added here as related? I feel we have some

Reply:
This rule is not about validation or error handling, that's just the example I used. 
It's about thinking of paths other than just the happy path

Reply:
I added one, but I dont think there are any close relations. Validation and errors are just an example, not really the topic of the rule.